src/Templates/NotificationBannerAdminPage.php

```diff
@@ -78,7 +78,7 @@ public function build_notifications_banner_page() {
 
 						<div class="bcgov-block-theme-body pb-0">
 
-							<h1 class="bcgov-block-theme-title" style="margin-bottom: 1rem;">Notification Banner Settings</h1>
+							<h1 class="bcgov-block-theme-title" style="margin-bottom: 1rem;">Custom Notice Banner Settings</h1>
```

Comment:
all text regardless if it is admin or public text, probably should be wrapped i18n function of some sort